myapp/db/schema.rb

```diff
@@ -10,63 +10,72 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema.define(version: 20_220_210_065_045) do
-  create_table 'boards', options: 'ENGINE=InnoDB DEFAULT CHARSET=utf8mb4', force: :cascade do |t|
-    t.string 'title'
-    t.datetime 'created_at', precision: 6, null: false
-    t.datetime 'updated_at', precision: 6, null: false
+ActiveRecord::Schema.define(version: 2022_02_28_005248) do
+
+  create_table "boards", options: "ENGINE=InnoDB DEFAULT CHARSET=utf8mb4", force: :cascade do |t|
+    t.string "title"
+    t.datetime "created_at", precision: 6, null: false
+    t.datetime "updated_at", precision: 6, null: false
+  end
```

Comment:
Rubyでは文字列よりもシンボルを積極的に使うことをお勧めします。
https://qiita.com/yyykms123/items/6a6ae7fe8cd9263a3d1c

```suggestion
  create_table "boards", options: "ENGINE=InnoDB DEFAULT CHARSET=utf8mb4", force: :cascade do |t|
    t.string :title
    t.datetime :created_at, precision: 6, null: false
    t.datetime :updated_at, precision: 6, null: false
  end
```

Reply:
@hayashiR1 ありがとうございます。資料のリンクも貼っていただいて、Rubyのシンボルに関して理解できました。修正しました。